cob_bringup/tools/docker_control.launch

```diff
@@ -32,4 +33,8 @@
 	<node pkg="cob_docker_control" type="docker_control.py" respawn="false" name="docker_control" output="screen" unless="$(arg multi_station)">
 	</node>
 
+	<!-- virtual charging and power usage for simulation -->
+	<node if="$(arg sim)" pkg="cob_docker_control" type="fake_docking.py" respawn="false" name="fake_docker_control" output="screen" >
+	</node> 
```

Comment:
Even I am not sure about the ``unless="$(arg sim)"`` tag, simulation works fine though.